### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> node ./src/scripts/update-icons.js

Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element

### CODE REVIEW

File: icons/folder-open-color.svg
Comment (fallaciousreasoning) [inline icons/folder-open-color.svg:1]: @aguscruiz purely my (unsolicited) opinion but I like the old ones better

Comment (aguscruiz) [reply icons/folder-open-color.svg:1]: Let me take a new stab at it, see if I can do anything different and better

Comment (aguscruiz) [reply icons/folder-open-color.svg:1]: I made an update. There was something that was really bugging me with the difference between the copy / paste relative icon sizes. I think this fixes it up pretty nicely

Comment (fallaciousreasoning) [reply icons/folder-open-color.svg:1]: :heart:

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**